data_management/opensearch_indexer/tests/conftest.py

```diff
@@ -75,6 +75,27 @@ class FileMetadata(Base):
     file = relationship("File", foreign_keys="FileMetadata.FileId")
 
 
+class FFIDMetadata(Base):
```

Comment:
we know we have this tech debt of repeating here and in the app models file, and have a ticket to consolidate, just need to prioritise